Error: prefix already exists: /home/ubuntu/miniconda/envs/_pipbuild_
Exception:
Traceback (most recent call last):
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/basecommand.py", line 223, in main
    status = self.run(options, args)
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/commands/install.py", line 268, in run
    wheel_cache
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/basecommand.py", line 268, in populate_requirement_set
    wheel_cache=wheel_cache
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/req/req_install.py", line 207, in from_line
    wheel_cache=wheel_cache)
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/req/req_install.py", line 66, in __init__
    req = pkg_resources.Requirement.parse(req)
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2960, in parse
    reqs = list(parse_requirements(s))
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2904, in parse_requirements
    "version spec")
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2869, in scan_list
    raise ValueError(msg, line, "at", line[p:])
ValueError: ('Expected version spec in', 'smmap==>=0.8.5', 'at', '==>=0.8.5')

Package search: {'_pypi_ordering': False, 'version': '0.6.4', 'name': 'gitdb', 'summary': 'Git Object Database'}
Creating standard recipe for gitdb-0.6.4
Converting recipe in /home/ubuntu/pypi-conda-builds/gitdb
package = gitdb
   dependencies = [u'smmap >=0.8.5']
Creating standard recipe for smmap->=0.8.5
Error: Version >=0.8.5 of smmap is not available on PyPI.

conda create -n _pipbuild_ --yes python pip
/home/ubuntu/miniconda/envs/_pipbuild_/bin/pip install smmap==>=0.8.5
Error: Could not pip install smmap==>=0.8.5
